src/seedu/addressbook/data/person/PrivatableContactDetail.java

```diff
@@ -18,4 +18,8 @@ public PrivatableContactDetail(String value, boolean isPrivate) throws IllegalVa
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    public String getPrintableString() {
+        return getPrintableStringPrefix() + (isPrivate() ? "(private) " : "") + this;
```

Comment:
why return `this` in the end? 
since from this commit i dunno what is the to string format, it is difficult to judge. but i assume you would have done it right! :-)